views/partials/leftSideInfo.pug

```diff
@@ -0,0 +1,19 @@
+div(class="left-container")
+  div(class="tabs highlighted" id="getAll" onclick="updateSelected('getAll')")
+    span(class="get") GET
+    span /warriors
+  div(class="tabs" id="getType" onclick="updateSelected('getType')")
```

Comment:
You've got a mismatch here. This says `/type/{typeId}` but the description says `http://localhost:1337/warriors/{typeId}`

Reply:
I have no idea why I didn't catch that. It's been rectified